tests/e2e-agents/.env.example

```
@@ -0,0 +1,5 @@
+ANTHROPIC_API_KEY=sk-ant-...
+BASE_URL=http://localhost:3000
+TEST_EMAIL=1@a.ca
+TEST_PASSWORD=browsertest123
+MODEL=claude-sonnet-4-20250514
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MODEL key should go before the TEST_EMAIL key

(UnorderedKey)